### DISCUSSION TITLE
acme.com/app instead of app.acme.com?

### DISCUSSION THREAD

**Discussion by coodoo** [vercel/vercel#5249] [Help] [answered]
I was reading [this post](https://vercel.com/blog/monorepos) about monorepo deployment and wondering is it possible to have three repos as three sub-route instead of sub-domains? Like this:

1. acme.com
2. acme.com/app
3. acme.com/blog

Instead of having three sub-domains as depicted below:
![](https://assets.vercel.com/image/upload/q_auto/v1598633015/front/blog/monorepos/light-arch.png)

I tried the `rewrite` trick in `vercel.json` but seemed there's no way to cloak/mask the url?

**Comment by mcsdevv** [comment, marked as answer]
Hey @coodoo, you can find information on how to do this here: https://vercel.com/knowledge/can-i-redirect-from-a-subdomain-to-a-subpath
